integrations/pytorch/README.md

```diff
@@ -80,12 +80,14 @@ The following table lays out the root-level files and folders along with a descr
-| notebooks                     | Jupyter notebooks to walk through sparsifiation within standard PyTorch training flows                                |
-| recipes                       | Typical recipes for sparsifying PyTorch models along with any downloaded recipes from the SparseZoo.                  |
-| tutorials                     | Tutorial walkthroughs for how to sparsify PyTorch models using recipes.                                               |
-| README.md                     | Readme file.                                                                                                          |
-| torchvision_sparsification.py | Example training script for sparsifying PyTorch torchvision classification models.                                    |
-| vision.py                     | Utility training script for sparsifying classification and detection models in PyTorch (loads SparseML models).       |
+| [notebooks](https://github.com/neuralmagic/sparseml/tree/main/integrations/pytorch/notebooks)                     | Jupyter notebooks to walk through sparsifiation within standard PyTorch training flows                                |
+| [recipes](https://github.com/neuralmagic/sparseml/tree/main/integrations/pytorch/recipes)                       | Typical recipes for sparsifying PyTorch models along with any downloaded recipes from the SparseZoo.                  |
+| [tutorials](https://github.com/neuralmagic/sparseml/tree/main/integrations/pytorch/tutorials)                     | Tutorial walkthroughs for how to sparsify PyTorch models using recipes.                                               |
+| [README.md](https://github.com/neuralmagic/sparseml/tree/main/integrations/pytorch/README.md)                     | Readme file.                                                                                                          |
+| [torchvision_sparsification.py](https://github.com/neuralmagic/sparseml/tree/main/integrations/pytorch/torchvision_sparsification.py) | Example training script for sparsifying PyTorch torchvision classification models.                                    |
+| [train.py](https://github.com/neuralmagic/sparseml/tree/main/integrations/pytorch/train.py)                     | Utility training script for sparsifying classification and detection models in PyTorch (loads SparseML models).       |
+| [export.py](https://github.com/neuralmagic/sparseml/tree/main/integrations/pytorch/export.py)                     | Utility script for exporting pytorch model to ONNX file along with sample inputs, outputs and labels.                 |
+|[analysis.py](https://github.com/neuralmagic/sparseml/tree/main/integrations/pytorch/analysis.py)|Utility script to Run a kernel sparsity (pruning) or learning rate sensitivity analysis for a desired image classification architecture |
```

Comment:
sorry to be nit-picky about it but I would suggest hyper-referencing relative paths, e.g.:

```
 [notebooks](./integrations/pytorch/notebooks)
```

This way it will auto-resolve to the correct branch depending on the branch you are on and IDEs like VSCode can also link to the right directory instead of going to the main branch on github.com

Reply:
Thanks @lucaslie didn't know about this
![](https://media3.giphy.com/media/xUPGcxpCV81ebKh7Vu/giphy.gif)

nit
`[notebooks](notebooks)`